Directory.Packages.props

```diff
@@ -78,6 +78,8 @@
     <PackageVersion Include="AspNetCore.HealthChecks.Redis" Version="9.0.0" />
     <PackageVersion Include="AspNetCore.HealthChecks.SqlServer" Version="9.0.0" />
     <PackageVersion Include="AspNetCore.HealthChecks.Uris" Version="9.0.0" />
+    <!-- MSBuild dependencies -->
+    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.9.5" /> <!-- This should not be updated, so we support older VS versions -->
```

Comment:
According to https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/setup-tooling we support VS 17.9